core/rfb.js

```diff
@@ -1018,6 +1021,93 @@ export default class RFB extends EventTargetMixin {
         return this._negotiate_authentication();
     }
 
+    // VeNCrypt authentication, currently only supports version 0.2 and only Plain subtype
+    // protocol: https://github.com/rfbproto/rfbproto/blob/617d72b4ced0da7d90d4a70cc6d6e1a30819f597/rfbproto.rst#vencrypt
```

Comment:
We don't need this as all of this file uses that specification as the reference.